internal/expanders/gatekeeper_test.go

```diff
@@ -0,0 +1,160 @@
+// // Copyright Contributors to the Open Cluster Management project
+package expanders
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/open-cluster-management/policy-generator-plugin/internal/types"
+)
+
+func TestGatekeeperCanHandle(t *testing.T) {
+	t.Parallel()
+	g := GatekeeperPolicyExpander{}
+
+	tests := []struct{ kind string }{
+		{"MyConstraint"},
+	}
+
+	for _, test := range tests {
+		t.Run(
+			fmt.Sprintf("kind=%s", test.kind),
+			func(t *testing.T) {
+				t.Parallel()
+				manifest := map[string]interface{}{
+					"apiVersion": gatekeeperConstraintAPIVersion,
+					"kind":       test.kind,
+					"metadata": map[string]interface{}{
+						"name": "my-awesome-constraint",
+					},
+				}
+				assertEqual(t, g.CanHandle(manifest), true)
+			},
+		)
+	}
+}
+
+func TestGatekeeperCanHandleInvalid(t *testing.T) {
+	t.Parallel()
+	g := GatekeeperPolicyExpander{}
+
+	tests := []struct{ apiVersion, kind, name string }{
+		{"v1", "MyConstraint", "my-awesomer-policy"},
```

Comment:
"my-awesomer-policy"... that's a nice subtle shot at my Kyverno unit test.  :laughing: